recipes/repo.rb

```diff
@@ -284,6 +284,102 @@
   end
 end
 
+# RPM repository setup
+apt_repository 'createrepo-agent-ppa' do
+  uri 'ppa:osrf/createrepo-agent'
+end
+
+package 'createrepo-agent'
+
+package 'createrepo-c'
+
+package 'rpm'
+
+package 'socat'
+
+node['ros_buildfarm']['rpm_repos'].each do |dist, versions|
+  dist_dir = "/var/repos/#{dist}"
+
+  %w(building testing main).each do |repo|
+    repo_dir = "#{dist_dir}/#{repo}"
+    file "#{repo_dir}/RPM-GPG-KEY-ros-#{repo}" do
+      action :delete
+    end
```

Comment:
What introduced the file that this is deleting? I don't see it on the main production host.

Reply:
This was a copy/paste, really. We originally hosted the public GPG key here but dropped it in favor of downloading it from Github. This was a pretty long time ago and we can probably just drop it.

Reply:
Got it. If we created that file in a tagged release of this cookbook I'd be inclined to keep it in for the next major bump as part of a formal deprecation but if it never made it into a release I don't think that's necessary.